routes/handlers/event.js

```diff
@@ -140,14 +140,36 @@ const create = {
   }
 };
 
-const createProximityFilter = function (fieldName, maxDistance, coord, postalCode) {
+const createProximityFilter = function (fieldName, query) {
```

Comment:
Its not a huge deal, but maybe we can merge all geo functionality in a single library. It's something I should have done, so no worries if you'd prefer to do it later or leave it for me to do.